create_reports.py

```diff
@@ -96,24 +97,39 @@ def create_daily_report(config):
         create_heatmap_report(config, yesterday_csv, violation_heatmap_file, 'Violations')
 
 
-def send_daily_report_notification(config, source):
+def send_daily_report_notification(config, entity_info):
+    entity_type = entity_info['type']
+    all_violations_per_hour = []
     log_directory = config.get_section_dict("Logger")["LogDirectory"]
-    objects_log_directory = os.path.join(log_directory, source['id'], "objects_log")
     yesterday = str(date.today() - timedelta(days=1))
-    daily_csv = os.path.join(objects_log_directory, 'report_' + yesterday + '.csv')
-    violations_per_hour = []
-    with open(daily_csv, newline='') as csvfile:
-        reader = csv.DictReader(csvfile)
-        for row in reader:
-            violations_per_hour.append(int(row['ViolatingObjects']))
+
+    if entity_type == 'Camera':
+        objects_log_directory = os.path.join(log_directory, entity_info['id'], "objects_log")
+        daily_csv_file_paths = [os.path.join(objects_log_directory, 'report_' + yesterday + '.csv')]
+    else:
+        # entity == 'Area'
+        camera_ids = entity_info['cameras']
+        daily_csv_file_paths = [os.path.join(log_directory, camera_id, "objects_log/report_" + yesterday + ".csv") for camera_id in camera_ids]
+        daily_csv_file_paths = daily_csv_file_paths + daily_csv_file_paths
```

Comment:
Why are you duplicating the file paths here?

Reply:
Nice catch! (I did it for testing purposes and forgot to remove it)